tests/core/test_views_errors.py

```diff
@@ -40,19 +40,16 @@ def test_backoff_header_is_present_on_error_responses(self):
             self.assertEquals(response.headers['Backoff'], '10')
 
     def test_404_is_valid_formatted_error(self):
-        response = self.app.get('/unknown', status=404)
-        self.assertFormattedError(response, 404, ERRORS.MISSING_RESOURCE, "Not Found",
-                                  "The resource you are looking for could not be found.")
```

Comment:
I thought we want to add the details attribute to all 404 errors as in the normal 404 should also contain a details attribute in the response.